packages/amplify-e2e-tests/schemas/simple_model.graphql

```diff
@@ -1,4 +1,9 @@
 type Todo @model {
   id: ID!
   content: String
+  contentSupport: [contentSupport]
```

Comment:
added an Embeddable type `contentSupport` to an existing Todo. Can remove this if suggested to do so